documentation/add/07-sqw_redesign.md

```diff
@@ -22,18 +22,40 @@ In addition, the extra information and raw data contained on the SQW object allo
 
 ## Main Classes
 
+![Class Overview](../diagrams/sqw-dnd.png)
+
+
+
+### aSQWDNDBase
+
+Abstract base class for the SQW and DND objects.
+
+This includes the image data and main header information, available in both the `SQW` and `DnD` classes.
+
+This class will also include common methods, in particular the large number of unary and binary operations which are implemented as calls to operations managers which will be defined in the implementing classes.
+
 ### SQW
 
 The `SQW` object is the providing the public API to data. Data manipulations are performed on the `PixelBlock` and `Image` is recalculated.
 
+This class includes the full experiment data including the raw pixel data and details of the instrument and detectors.
+
 ![SQW Class Overview](../diagrams/sqw.png)
 
-### DND
+### aDND, DnD
+
+The `DnD` object is a "cut-down SQW" object containing only `Image` data.  
+
+This exists in n-dimensional forms, with each class extending an abstract base class The `PixelBlock`,  `IX_Instr`and `IX_DetectorArray` information is NOT included, and any data manipulation operations are performed directly on the `Image` data.
```

Comment:
Missing full-stop after "abstract base class".

Also, might be worth splitting some of these longer lines into several lines. I don't stick to this religiously, but using [semantic line breaks](https://sembr.org/) does tend to make reviewing Markdown easier